force-app/main/default/lwc/takeHomePayCalculator/takeHomePayCalculator.html

```diff
@@ -0,0 +1,75 @@
+<template>  
+    <lightning-card title="Take Home Pay Calculator">
+        <lightning-button 
+            variant="brand" 
+            label="Calculate" 
+            title="Calculate" 
+            slot="actions" 
+            onclick={calculateTakeHomePay}
+        ></lightning-button>
+        <div class="slds-var-p-horizontal_small">
+            <lightning-input
+                type="number"
+                name="Salary"
+                label="Salary"
+                value={SalaryValue}
+                onchange={handleSalaryChange}
+            ></lightning-input>
+            <lightning-combobox
+                name="Pay Frequency"
+                label="Pay Frequency"
+                value={selectedOption}
+                placeholder="Select Pay Frequency"
+                options={options}
+                onchange={handleChange}
+            ></lightning-combobox>
+            <lightning-radio-group 
+                name="radioGroup"
+                label="Marital Status"
+                options={radioOptions}
+                value={value}
+                type="radio"
+                onchange={handleRadioChange}
+            ></lightning-radio-group>
+            <div style="margin-bottom: 20px;"></div>
```

Comment:
This might be best moved to a css file or maybe there is something in the SLDS you can leverage 🙂